src/plugins/fakeNitro.ts

```diff
@@ -392,70 +393,139 @@ export default definePlugin({
         });
     },
 
-    patchFakeNitroEmojisOrRemoveStickersLinks(content: Array<any>, inline: boolean) {
-        if (content.length > 1 && !settings.store.transformCompoundSentence) return content;
+    trimContent(content: Array<any>) {
+        const firstContent = content[0];
+        if (typeof firstContent === "string") content[0] = firstContent.trimStart();
+        if (content[0] === "") content.splice(0, 1);
 
-        const newContent: Array<any> = [];
+        const lastIndex = content.length - 1;
+        const lastContent = content[lastIndex];
+        if (typeof lastContent === "string") content[lastIndex] = lastContent.trimEnd();
+        if (content[lastIndex] === "") content.length -= 1;
+    },
 
-        let nextIndex = content.length;
+    clearEmptyArrayItems(array: Array<any>) {
+        const newArray: Array<any> = [];
```

Comment:
```suggestion
        return array.filter(Boolean)
```
(kinda obsolete function)